src/main/java/jarvis/logic/Jarvis.java

```diff
@@ -0,0 +1,59 @@
+package jarvis.logic;
+import java.util.Scanner;
+
+public class Jarvis {
+        TaskList tasklist;
+        Parser parser;
+        Storage storage;
+
+        public Jarvis(){
+            this.tasklist = new TaskList();
+            this.parser = new Parser();
+            this.storage = Storage.getInstance();
+        }
+
```

Comment:
I believe this is one level indentation too many, maybe shift the indentation to the left?